spark/dpl-interpreter/pom.xml

```diff
@@ -0,0 +1,452 @@
+  </parent>
+  <artifactId>pth_07</artifactId>
+  <packaging>jar</packaging>
+  <version>${revision}${sha1}${changelist}</version>
+  <name>Zeppelin: Teragrep DPL Interpreter</name>
+  <description>Support for DPL in Zeppelin</description>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+    <java.version>11</java.version>
+    <revision>0.0.1</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <sha1/>
+    <interpreter.name>dpl</interpreter.name>
+    <teragrep.pth_10.version>8.3.1</teragrep.pth_10.version>
+  </properties>
+  <scm>
+    <connection>scm:git:https://github.com/teragrep/pth_07.git</connection>
+    <developerConnection>scm:git:git@github.com:teragrep/pth_07.git</developerConnection>
+    <url>https://github.com/teragrep/pth_07/tree/master</url>
+  </scm>
+  <dependencies>
+    <!-- spark-core includes a version too old -->
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-all</artifactId>
+      <version>4.1.95.Final</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-buffer</artifactId>
+      <version>4.1.95.Final</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.zeppelin</groupId>
+      <artifactId>spark-interpreter</artifactId>
+      <version>${revision}${sha1}${changelist}</version>
+      <scope>provided</scope>
+    </dependency>
+    <!-- spark -->
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-core_2.12</artifactId>
+      <version>3.4.0</version>
+      <scope>provided</scope>
+      <exclusions>
+        <!-- exclude old netty lib, newer dep version above -->
+        <exclusion>
+          <groupId>io.netty</groupId>
+          <artifactId>netty-all</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-sql_2.12</artifactId>
+      <version>3.4.0</version>
+      <scope>provided</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-client</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-streaming_2.12</artifactId>
+      <version>3.4.0</version>
+      <scope>provided</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-client</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <!--DPL-dependencies -->
+    <dependency>
+      <groupId>com.teragrep</groupId>
+      <artifactId>pth_10</artifactId>
+      <version>${teragrep.pth_10.version}</version>
+    </dependency>
```

Comment:
should have scope provided so it will not be included as a jar into the interpreter itself

Reply:
Implemented and functional with https://github.com/teragrep/pth_10/pull/505 as far as I can tell